help_to_heat/frontdoor/urls.py

```diff
@@ -3,6 +3,7 @@
 from . import views
 
 frontdoor_patterns = [
+    path("thankyou", views.holding_page_view, name="holding-page"),
```

Comment:
is this duplicating line 8?

Reply:
Ah, yes! (Seems to just be some weirdness from rebasing :D)